brewtils/models.py

```diff
@@ -399,10 +404,12 @@ def is_different(self, other):
 class RequestFile(BaseModel):
     schema = "RequestFileSchema"
 
-    def __init__(self, storage_type=None, filename=None, id=None):
+    def __init__(
+        self, storage_type=None, filename=None, id=None  # noqa # shadows built-in
+    ):
         self.storage_type = storage_type
         self.filename = filename
-        self.id = id
+        self.id = id  # noqa # shadows built-in
```

Comment:
Not sure how to think about this: On the one hand, it makes perfect sense to have `id` and `type` as the names of member variables since they mark IDs and types of things. But on the other hand, they're protected keywords and these declarations are shadowing them (though it isn't clear that there'd ever be a need for the system version of them). It's a *bad* habit to shadow built-ins, but, for the moment, this suffices to shut linters up about it

Reply:
Just making the linter be quiet about this is the way to go. The amount of pain that would be required to change this, even to `id_`, would be enormous. It would require a major release and a deprecation cycle at the very least.